src/edu/csus/ecs/pc2/core/imports/clics/EventFeed.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+
+//@JsonIgnoreProperties(ignoreUnknown = true)
+public class EventFeed {
+
+    // {"type": "teams", "id": "289de609-b1a0-40d8-a9e0-54a30e752e69", "op": "create", "data": {"id": "169347", "name": "Colin Peppler", "icpc_id": null, "group_ids": [], "display_name": "Virginia
+    // Tech", "organization_id": "vt_edu"}}
+
+    @JsonProperty
+    private String type;
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String op;
+
```

Comment:
"op" is not defined in the 2202-07 spec (although it was defined in earlier spec versions).  Is this field here for backward-compatibility?  (If so, it seems like there's a lot of OTHER "backward-compatibility" stuff missing...)

Reply:
No, this effort was not intended to bring all of pc2 up to spec, any spec.   To note that op should be removed
add that to #559.

Reply:
Since op is not harming anything, if that should be removed - add a new Issue.
Working to not have a single issue that "does it all".